### ISSUE TITLE
term info should return classification

### ISSUE THREAD

**Issue by balhoff** [phenoscape/phenoscape-kb-services#413]
Also, check how object property relations are queried—possibly change to using relation graph.

**Comment by balhoff** [comment]
@hlapp I'm implementing this and wondering if you want subclasses in this; or instead just superclasses and equivalents.

**Comment by balhoff** [comment]
One more question: I was planning to merge the classification into relationships. Let me know if this is not desired. Here's an example:

```json
{
  "synonyms": [
    {
      "property": "http://www.geneontology.org/formats/oboInOwl#hasExactSynonym",
      "value": "basihyoid"
    },
    {
      "property": "http://www.geneontology.org/formats/oboInOwl#hasExactSynonym",
      "value": "glossohyal"
    }
  ],
  "label": "basihyal bone",
  "relationships": [
    {
      "property": {
        "@id": "http://www.w3.org/2000/01/rdf-schema#subClassOf",
        "label": "subclass of"
      },
      "value": {
        "@id": "http://purl.obolibrary.org/obo/UBERON_0002513",
        "label": "endochondral bone"
      }
    },
    {
      "property": {
        "@id": "http://www.w3.org/2000/01/rdf-schema#subClassOf",
        "label": "subclass of"
      },
      "value": {
        "@id": "http://purl.obolibrary.org/obo/UBERON_0011614",
        "label": "basihyal element"
      }
    },
    {
      "property": {
        "@id": "http://purl.obolibrary.org/obo/RO_0002202",
        "label": "develops from"
      },
      "value": {
        "@id": "http://purl.obolibrary.org/obo/UBERON_0011615",
        "label": "basihyal cartilage"
      }
    }
  ],
  "isDefinedBy": "http://purl.obolibrary.org/obo/uberon.owl",
  "definition": "Replacement bone that is median and is the anterior-most bone of the ventral hyoid arch.",
  "@id": "http://purl.obolibrary.org/obo/UBERON_0011618"
}
```

**Comment by hlapp** [comment]
The classification endpoint returns `superClassOf` so this one should too, I think. (Otherwise we'd still have to possibly query the classification endpoint.)

Also, it'd be good to maintain the structure returned by `classification`, i.e., return `subClassOf`, `equivalentTo`, and `superClassOf` separately from relationships. Otherwise we'll need to hardcode which relationships are "special" (i.e., built into OWL) versus part of the data model representation in  OWL. We do treat these separately in Rphenoscape in the `terminfo` class, and I think we'll want to keep doing that (because otherwise we'd be punting that task to the Rphenoscape user, which feels counter to Rphenoscape mission).

**Comment by balhoff** [comment]
Okay, thanks!

**Comment by balhoff** [comment]
@hlapp here is an updated structure:

```json
{
  "synonyms": [
    {
      "property": "http://www.geneontology.org/formats/oboInOwl#hasExactSynonym",
      "value": "basihyoid"
    },
    {
      "property": "http://www.geneontology.org/formats/oboInOwl#hasExactSynonym",
      "value": "glossohyal"
    }
  ],
  "label": "basihyal bone",
  "relationships": [
    {
      "property": {
        "@id": "http://purl.obolibrary.org/obo/RO_0002202",
        "label": "develops from"
      },
      "value": {
        "@id": "http://purl.obolibrary.org/obo/UBERON_0011615",
        "label": "basihyal cartilage"
      }
    }
  ],
  "isDefinedBy": "http://purl.obolibrary.org/obo/uberon.owl",
  "definition": "Replacement bone that is median and is the anterior-most bone of the ventral hyoid arch.",
  "classification": {
    "subClassOf": [
      {
        "@id": "http://purl.obolibrary.org/obo/UBERON_0011614",
        "label": "basihyal element"
      },
      {
        "@id": "http://purl.obolibrary.org/obo/UBERON_0002513",
        "label": "endochondral bone"
      }
    ],
    "superClassOf": [
      {
        "@id": "http://purl.obolibrary.org/obo/ZFA_0000316",
        "label": "basihyal bone (zebrafish)"
      }
    ],
    "equivalentTo": [

    ]
  },
  "@id": "http://purl.obolibrary.org/obo/UBERON_0011618"
}
```

**Comment by balhoff** [comment]
Fixed by #419.
